sbi/src/sbi.rs

```diff
@@ -738,6 +738,35 @@ pub enum TeeFunction {
     ///
     /// a6 = 15
     TsmLocalFence,
+    /// Defines the single GPA range that will be used to map non-confidential shared pages.
+    /// The call made not be made after TVM is finalized, and the pages in the range cannot overlap
+    /// with the GPAs for confidential pages
+    /// a6 = 17
+    TsmDefineSharedPagesRange {
```

Comment:
Two things:
- The `Tvm` prefix would be more appropriate here since it's an operation on a TVM rather than the TSM itself.
- Limiting the TVM to one shared page range seems a bit overly restrictive, at least for a public ABI (the implementation of course could refuse to add more than one).